spec/unit/indirector/catalog/compiler_spec.rb

```diff
@@ -414,6 +414,30 @@ def a_request_that_contains(facts)
     end
   end
 
+  describe "after finding nodes, in regard to the server's Puppet Enterprise version" do
+    pe_version_file = '/opt/puppetlabs/server/pe_version'
```

Comment:
could you convert this to `let(:pe_version_file)`?

Reply:
Sorry, yes!